OpenAPI/OpenAPI.NET/overview.md

```diff
@@ -29,6 +29,6 @@ Some of the benefits of using the OpenAPI.NET library when working with OpenAPI
 
 ## See also
 
-- [Creating an OpenAPI document](create-openapi.md)
```

Comment:
those need to be relative links for the docs fx to work.
I don't know why we have two folders with different casing for the same name. This is probably what should be addressed instead.